src/prefect/cli/deploy.py

```diff
@@ -359,8 +368,7 @@ async def _run_single_deploy(
             raise ValueError(
                 "An entrypoint must be provided:\n\n"
                 " \t[yellow]prefect deploy path/to/file.py:flow_function\n\n"
-                "You can also provide an entrypoint in a prefect.yaml"
-                " file located in the current working directory."
+                "You can also provide an entrypoint in a prefect.yaml file."
```

Comment:
Does this still have to be `prefect.yaml`?

Reply:
Nope, it can technically be named anything we want, but I think naming it something like `prefect.yaml` would be best practice. I can imagine a useful standard being `dev_prefect.yaml` `prod_prefect.yaml`.

Reply:
I see, so we're just nudging them in the right direction here, got it.